src/python/pants/backend/python/rules/python_test_runner.py

```diff
@@ -43,6 +65,22 @@ def calculate_timeout_seconds(
   return target_timeout
 
 
+def get_packages_to_cover(
+  test_target: PythonTestsAdaptor,
+  coverage: str,
+  source_root_stripped_file_paths: List[str],
+) -> Set[str]:
+  if coverage == 'auto':
```

Comment:
What are the acceptable values here? Can you add some docstring?

Reply:
Sure. This comes directly from the `--coverage` option, which mentions this magic value, but I can add a docstring here too.